website/source/docs/agent/config-entries/service-router.html.md

```diff
@@ -109,124 +131,128 @@ routes = [
   stops further evaluation. Traffic that fails to match any of the provided
   routes will be routed to the default service.
 
-  - `Match` `(ServiceRouteMatch: <optional>)` - A set of criteria that can
-    match incoming L7 requests. If empty or omitted it acts as a catch-all.
+    - `Match` `(ServiceRouteMatch: <optional>)` - A set of criteria that can
+      match incoming L7 requests. If empty or omitted it acts as a catch-all.
+
+        - `HTTP` `(ServiceRouteHTTPMatch: <optional>)` - A set of
+          [http-specific match criteria](#serviceroutehttpmatch).
+
+    - `Destination` `(ServiceRouteDestination: <optional>)` - Controls [how to
+      proxy](#serviceroutedestination) the actual matching request to a
+      service.
+
+### `ServiceRouteHTTPMatch`
```

Comment:
There must be some middleman maximum line length before weirdness happens. If I left this as one (_correctly_ indented) list-of-list-of-lists-of-... two of the three links to the envoy regex documentation render with the link text omitted which is super weird.

Breaking this up reduces the maximum indentation distance, which has the added benefit of making this easier to read.